com.chartboost.mediation/Runtime/ChartboostMediation.cs

```diff
@@ -0,0 +1,228 @@
+
+        /// <inheritdoc cref="IChartboostMediationRewardedEvents.DidCloseRewarded"/>>
+        public static event ChartboostMediationPlacementEvent DidCloseRewarded
+        {
+            add => _chartboostMediationExternal.DidCloseRewarded += value;
+            remove => _chartboostMediationExternal.DidCloseRewarded -= value;
+        }
+
+        /// <inheritdoc cref="IChartboostMediationRewardedEvents.DidClickRewarded"/>>
+        public static event ChartboostMediationPlacementEvent DidClickRewarded
+        {
+            add => _chartboostMediationExternal.DidClickRewarded += value;
+            remove => _chartboostMediationExternal.DidClickRewarded -= value;
+        }
+        
+        /// <inheritdoc cref="IChartboostMediationRewardedEvents.DidRecordImpressionRewarded"/>>
+        public static event ChartboostMediationPlacementEvent DidRecordImpressionRewarded
+        {
+            add => _chartboostMediationExternal.DidRecordImpressionRewarded += value;
+            remove => _chartboostMediationExternal.DidRecordImpressionRewarded -= value;
+        }
+        
+        /// <inheritdoc cref="IChartboostMediationRewardedEvents.DidReceiveReward"/>>
+        public static event ChartboostMediationPlacementEvent DidReceiveReward
+        {
+            add => _chartboostMediationExternal.DidReceiveReward += value;
+            remove => _chartboostMediationExternal.DidReceiveReward -= value;
+        }
+        #endregion
+
+        #region Banner Callbacks
+        /// <inheritdoc cref="IChartboostMediationBannerEvents.DidLoadBanner"/>>
+        public static event ChartboostMediationPlacementLoadEvent DidLoadBanner
+        {
+            add => _chartboostMediationExternal.DidLoadBanner += value;
+            remove => _chartboostMediationExternal.DidLoadBanner -= value;
+        }
+
+        /// <inheritdoc cref="IChartboostMediationBannerEvents.DidClickBanner"/>>
+        public static event ChartboostMediationPlacementEvent DidClickBanner
+        {
+            add => _chartboostMediationExternal.DidClickBanner += value;
+            remove => _chartboostMediationExternal.DidClickBanner -= value;
+        }
+        
+        /// <inheritdoc cref="IChartboostMediationBannerEvents.DidRecordImpressionBanner"/>>
+        public static event ChartboostMediationPlacementEvent DidRecordImpressionBanner
+        {
+            add => _chartboostMediationExternal.DidRecordImpressionBanner += value;
+            remove => _chartboostMediationExternal.DidRecordImpressionBanner -= value;
+        }
+        #endregion
+
+        //////////////////////////////////////////////////////
+        // Functions for showing ads
+        //////////////////////////////////////////////////////
+
+        /// <summary>
+        /// Returns a new ad unit that can be used to load and display interstitial ads.
+        /// </summary>
+        /// <param name="placementName">The placement ID for the Chartboost Mediation impression type.</param>
```

Comment:
I think these are now going to be called Chartboost placements. But, we should be ok for now.